packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -149,7 +149,13 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
         if (!communicator.listeners('uncaught:error').length) {
           communicator.once('uncaught:error', ({ err }) => {
             // @ts-ignore
-            Cypress.runner.onSpecError('error')({ error: err })
+            if (err.name === 'CypressError') {
+              // This is a Cypress error thrown from the secondary domain, do not wrap it.
```

Comment:
I wonder if it makes sense to clarify here that this only handles cypress errors thrown after the queue/context window is closed as other Cypress errors are handled differently, as well as what is not being wrapped.

```suggestion
              // This is a Cypress error thrown from the secondary domain after the command queue has finished, do not wrap it as a spec or app error.
```